external/fv3fit/fv3fit/keras/_models/dense.py

```diff
@@ -117,17 +122,22 @@ def train_dense_model(
     )
     del train_batches
 
+    loss_history = TrainingLoopLossHistory()
     hyperparameters.training_loop.fit_loop(
-        model=train_model, Xy=train_data, validation_data=validation_data
+        model=train_model,
+        Xy=train_data,
+        validation_data=validation_data,
+        callbacks=[loss_history.callback],
     )
     predictor = PureKerasModel(
         input_variables=hyperparameters.input_variables,
         output_variables=hyperparameters.output_variables,
         output_metadata=output_metadata,
         model=predict_model,
-        unstacked_dims=("z",),
+        unstacked_dims=("z"),
```

Comment:
Can you revert this? While technically this code will run, the meaning changes. previously it was a tuple containing "z" as its one element, now it is passing the string "z" as the value for unstacked_dims. This works because if you iterate over a string it will return each character, but this will break as soon as the unstacked dimension is more than one character long.
```suggestion
        unstacked_dims=("z",),
```